Model/Sync/Orders/Processor.php

```diff
@@ -418,12 +422,14 @@ public function getOrderCollection($retryOrderIds = [])
      * @param Order $order
      * @param bool $isYotpoSyncedOrder
      * @param array<mixed> $yotpoSyncedOrders
+     * @param bool $realTImeSync
      * @return array<mixed>|DataObject
      * @throws LocalizedException
      * @throws NoSuchEntityException
      */
-    public function syncOrder($order, $isYotpoSyncedOrder, $yotpoSyncedOrders)
+    public function syncOrder($order, $isYotpoSyncedOrder, $yotpoSyncedOrders, $realTImeSync = false)
```

Comment:
everything that is related to `realTImeSync` shouldn't exist anymore.
please make sure that your code is aligned.

Reply:
This is done. Will update the PR soon